app/views/gitter/chat_button.erb

```diff
@@ -0,0 +1,17 @@
+<% if control.language.downcase == 'ruby' %>
+  <div class="gitter-chat-button-frame">
+    <div class="gitter-chat-button">
+      Open Chat: Enjoy the <%= control.language.capitalize %> conversation!
```

Comment:
This isn't going to work for languages like `Objective-C` or `ECMAScript` or `F#` where the track ID and the language name are not capitalized/downcased versions of each other.

The track data that has the `gitter` value, includes both the language name and the track id.